sympy/core/basic.py

```diff
@@ -1034,21 +1034,30 @@ def subs(self, *args, **kwargs):
         from .containers import Dict
         from .symbol import Dummy, Symbol
         from .numbers import _illegal
+        from .relational import Equality
 
         unordered = False
         if len(args) == 1:
-
             sequence = args[0]
             if isinstance(sequence, set):
                 unordered = True
             elif isinstance(sequence, (Dict, Mapping)):
                 unordered = True
                 sequence = sequence.items()
+            elif isinstance(sequence, Equality):
+                if isinstance(sequence.lhs, Symbol) and len(sequence.lhs.free_symbols) == 1:
+                    sequence = [[sequence.lhs, sequence.rhs]]
```

Comment:
```suggestion
                if isinstance(sequence.lhs, Symbol):
                    sequence = sequence.args
```

Reply:
Oh wait, we also need to wrap the args in a list.